src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
There is an unnecessary empty line at the beginning of the class implementation. Please remove it to adhere to the coding standards.